core/src/task.cpp

```diff
@@ -195,10 +195,14 @@ void Task::loadRobotModel(const std::string& robot_description) {
 
 void Task::add(Stage::pointer&& stage) {
```

Comment:
`Task::add` and `Task::insert` should be simple `using` forwards. There is no need to provide a new (very similar) implementation, isn't it?

Reply:
That's sadly not possible because of the additional Wrapper.
`Task` is actually a `WrapperBase`, but forwards `add` and `insert` to the wrapped stage.

That's why I considered, but rejected, to remove the Wrapper inheritance altogether.

Reply:
I see. But the new implementations should just forward to the corresponding methods of the wrapped stage then (instead of reimplementing the same logic again).

Reply:
True, I addressed it.